alvr/server_openvr/src/lib.rs

```diff
@@ -121,17 +121,21 @@ extern "C" fn driver_ready_idle(set_default_chap: bool) {
                         .unwrap_or(false);
 
                     if let Some(context) = &*SERVER_CORE_CONTEXT.read() {
+                        let target_timestamp =
+                            sample_timestamp + context.get_motion_to_photon_latency();
```

Comment:
Once this code actually gets enabled it will produce tracking timestamps past the actual render timestamp of steamvr. Combine this with a lack of timestamp sync and SteamVR might misbehave

Reply:
No, it will not. Currently the timestamps are already predicted, aka target timestamp. After new tracking negotiation, the timestamps will be sample timestamp. The target timestamp (and controller timestamp) are only used to predict the pose. It doesn't affect the amount that steamvr predicts

Reply:
Let me know if my reasoning is sound. It's night time in here

Reply:
Uuh it depends on what the sampletimeoffset (or we it's called) is set to. If we're gonna do it like this we should set it to 0